### DISCUSSION TITLE
Feature Inquiry - Simultaneous pcap Capture for Multiple SSL Libraries

### DISCUSSION THREAD

**Discussion by douglasmakey** [gojue/ecapture#449] [General]
Hello ecapture Team,

Firstly, I want to express my gratitude for the amazing work you've done with ecapture. It's an impressive project that has caught our attention.

**Comment by cfc4n** [comment]
Sorry, I forgot to reply. 

In the early versions, this feature was supported. However, it was later decoupled and split into separate modules. Currently, eCapture does not support capturing all SSL libraries simultaneously, and it is unclear in which scenarios this would be necessary. 

Nevertheless, you can modify the source code yourself to achieve this.

https://github.com/gojue/ecapture/blob/v0.6.3/cli/cmd/tls.go#L99-L158
